### ISSUE TITLE
abortError : Transaction aborted

### ISSUE THREAD

**Issue by Akta3d** [dfahlander/Dexie.js#529]
Sometimes I have error during put data in indexedDB.

What can cause an abort error, and why a transaction is implicitly used.

```
db.table('document').put(documentToSave)
.then(
   function ()
   {
    //...
   }
)
.catch(
function (err)
 {
logger.error('Fail to add document in idb. Error: ' + err);
}
);
```

Data to insert/update seems to be good

**Comment by valishah** [comment]
Hi , Even i have same issue. I found this when i manually cleared some tables and try to reinitialize the DB.

**Comment by Akta3d** [comment]
I don't succes to reproduce on my pc or tablet, but some of my users have this error.
I don't think they clear theirs DB.

I have this error on all browsers

**Comment by dfahlander** [comment]
May happen when a device has low disk space. See #323. A fix for this particular case (and actually any strange abort case) was merged into master and will be included in next beta release.

**Comment by glade-at-gigwell** [comment]
hey @dfahlander was the strange abort fix released?  With Chrome 58, we've seen a sharp increase in the number of users who are blocked from being able to use IndexDB/Dexie even after manually clearing the files off the disk under chrome.

**Comment by dfahlander** [comment]
Yes, Chrome will still abort when out of disk. The good news in [beta11](https://github.com/dfahlander/Dexie.js/releases/tag/v2.0.0-beta.11) is that you should see what real error actually caused this, and not just AbortError.

**Comment by vousys** [comment]
Hi guys I'm having the same issue only over windows10 and chrome61.
